hazelcast/src/main/java/com/hazelcast/query/impl/Index.java

```diff
@@ -109,6 +111,14 @@
      */
     Set<QueryableEntry> evaluate(Predicate predicate);
 
+    Iterator<QueryableEntry> getRecordIterator();
```

Comment:
Javadoc for the new methods would be useful.

Reply:
Added JavaDoc